Sources/PrimerSDK/Classes/Core/Primer/Primer.swift

```diff
@@ -167,12 +167,6 @@ public class Primer {
                 self?.root = RootViewController()
                 guard let root = self?.root else { return }
                 let router: RouterDelegate = DependencyContainer.resolve()
-                let settings: PrimerSettingsProtocol = DependencyContainer.resolve()
-                
-                if flow.internalSessionFlow.vaulted {
```

Comment:
What's happening here?

Reply:
Those lines were nullifying the amount in the vault flow, leading to a bug if you opened the vault flow and then the checkout flow. We don't need to nullify the amount (as it is one of the variables configured by the dev), we need to not present it in the vault flow.

Reply:
makes sense 👍